README.md

```diff
@@ -2,18 +2,65 @@
 
 # New Relic Monitoring for .NET
 
-You are viewing a previous version of the New Relic .NET agent repository. While the code here should be buildable, it may require some manual modifications or additional tooling in order to do so. This previous version is primarily for reference purposes.
+#### .NET Agent
+New Relic's .NET agent monitors your .NET app, giving you an end-to-end view of your app's performance. It works with all .NET compatible languages, such as C#, VB.NET and CLI.
+
+Note: this branch, `net35/main`, is maintained for backwards compatibility with .NET Framework 3.5 and corresponds to the 6.x versions of the New Relic .NET agent.  It primarily only recieves security patches and critical bug fixes.
 
 ## Support
 
-New Relic hosts and moderates an online forum where customers can interact with New Relic employees as well as other customers to get help and share best practices. Like all official New Relic open source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this project's topic/threads here:
+## Installation
+
+See our [agent installation documentation](https://docs.newrelic.com/docs/agents/net-agent/installation/introduction-net-agent-install).
+
+## Getting Started
+
+* See our [introduction to New Relic for .NET](https://docs.newrelic.com/docs/agents/net-agent/getting-started/introduction-new-relic-net) to learn how to use the .NET agent with your application.
+* See our [changelog](src/Agent/CHANGELOG.md) for release notes.
```

Comment:
should we be maintaining a separate CHANGELOG for this branch - this link refs the main branch.